components/ButtonBar/src/MDCButtonBarButton.h

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MaterialButtons.h"
+
+@interface MDCButtonBarButton : MDCFlatButton
+
+/**
+ Content padding for the button.
+ */
+@property(nonatomic) UIEdgeInsets contentPadding UI_APPEARANCE_SELECTOR;
+
+/**
+ The font used by the button's @c title.
+
+ If left unset or reset to nil for a given state, then a default font is used.
+
+ @param font The font.
+ @param state The state.
+ */
+- (void)setTitleFont:(nullable UIFont *)font forState:(UIControlState)state
```

Comment:
You shouldn't need this, it's already declared in MDCButton.h

Reply:
Understood.  It is intentionally redeclared to make it obvious that it can be customized.  If we don't think this is necessary I'm happy to remove.
However, when I re-declare, the compiler throws an error unless I re-define.  In this case the implementation just calls super.

Reply:
I suspect this is simply because once an interface is declared, it should be defined. In the case that the superclass removes its declaration, your interface remains the same.  If you prefer to add these into the header strictly for documentation purposes, then I think using a category would be the best way.  It allows redeclaring the API, but would not be enforced by the compiler as long as the category has no implementation.  The drawback would be that the compiler won't notice if the superclass drops its declaration.